contracts/MerkleDelegation.sol

```diff
@@ -71,22 +55,30 @@ contract MerkleDelegation is Ownable, Pausable {
         _unpause();
     }
 
-    function getDelegateRoot(address delegator)
+    function getDelegate(address delegator)
         external
         view
-        returns (bytes32 trieRoot)
+        returns (
+            address delegatorAddress,
```

Comment:
ah I wouldn't return this value as whoever is calling the function would know it already.

Reply:
Makes sense, addressed in [050a2c4](https://github.com/windranger-io/windranger-mass-delegation-contracts/pull/17/commits/050a2c488ffe845a5a0a0c7e761211e53772d9ee)